engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -342,7 +342,8 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
   @tailrec
   private def startRunnableScopes(data: WorkflowExecutionActorData): WorkflowExecutionActorData = {
     val runnableScopes = data.executionStore.runnableScopes.toList
-    val runnableCalls = runnableScopes.view collect { case k if k.scope.isInstanceOf[Call] => k } sortBy { _.index.getOrElse(-1) } map { _.tag }
+    val runnableCalls = runnableScopes.view collect { case k if k.scope.isInstanceOf[Call] => k } sortBy {
+      _.scope.fullyQualifiedName } sortBy { _.index.getOrElse(-1) } map { _.tag }
```

Comment:
I'm not sure 2 `sortBy`s in a row keep the ordering of the first one. I usually do 
```scala
sortBy { r => (r.scope.fullyQualifiedName, r.index.getOrElse(-1)) }
```